tests/h/search/index_test.py

```diff
@@ -179,8 +183,10 @@ def test_you_can_make_aggregations_on_tags_raw(self, es_client, factories, index
 
         response = search.execute()
 
-        tag_bucket_1 = next(bucket for bucket in response.aggregations.tags_raw_terms.buckets if bucket["key"] == "Hello")
-        tag_bucket_2 = next(bucket for bucket in response.aggregations.tags_raw_terms.buckets if bucket["key"] == "hello")
+        tag_bucket_1 = next(bucket for bucket in response.aggregations.tags_raw_terms.buckets if
```

Comment:
These indentation changes are not related to the rest of the PR. These lines were a bit long but I would prefer the change to be in a separate commit or PR.